### ISSUE TITLE
Feedback on "Navigate Chrome DevTools with Assistive Technology"

### ISSUE THREAD

**Issue by baldwin47** [GoogleChrome/developer.chrome.com#695]
https://developer.chrome.com/docs/devtools/accessibility/navigation/

I'm working with a screen reader user who is blind. His ambition is to be a developer and that entails using the DevTools in Chrome. This is a nicely written article but there seems to be a key flaw or something we aren't understanding. 

In [this section on inspecting elements](https://developer.chrome.com/docs/devtools/accessibility/navigation/#inspect) we're unable to actually reach a specific element in the Elements tab by inspection. Sending a right click from the screen reader as suggested opens up the elements tab as expected, but from that point things go unexpectedly:

1. In JAWS/NVDA i'm observing that the elements tab "focus" (highlight) lands on an element other than the intended one, often a parent div way up the tree (rather than say, a child span within a button i'm trying to inspect, which would be okay)
2. On Mac Voiceover I'm observing that elements tab "focus" *sometimes* lands on the correct element, but i'm unable to get the screen reader to read that element out. It seems this has something to do with the painful VO "enter the group" behavior. But, once I press VO + shift + down arrow, I hear the "entering group" noise, but I am at the top of the DOM. 

Similarly, if we attempt to search for an element in the DOM (seems like this would be a decent workaround), focus isn't managed to the element found/highlighted by search. 

So, with these two issues I'm not sure how to reach a specific element other than by painstaking linear navigation of the DOM. 

Is this a known issue, is the behavior now different from when the article was written in 2019, or are we doing something wrong here? I'm not sure how a screen reader user would effectively make use of the Elements tab without the ability to target specific nodes in the DOM.

**Comment by robdodson** [comment]
tentatively assigning to myself, but also reaching out to the devtools team to see who (if anyone) still owns screen reader support. The previous owner has left google.

**Comment by dstmarthe** [comment]
Thanks for raising this. We’re moving to a new platform and so we’ll fix this over there. I’ve moved the issue to our new tracker and you can see it at: https://issuetracker.google.com/issues/311795733
